docs/docusaurus/docs/reference/learn/data_quality_use_cases/distribution.md

```diff
@@ -1,8 +1,297 @@
+
+### Expect Column Value Z Scores To Be Less Than
+
+Checks that the Z-scores (number of standard deviations from mean) of all values are below a threshold.
+
+**Use Case**: Powerful for identifying individual outliers and anomalous data points that could represent data entry issues or unusual transactions.
+
+```python
+gxe.expect_column_value_z_scores_to_be_less_than(
+    column="purchase_amount",
+    threshold=3,
+)
+```
+
+<small>View `ExpectColumnValueZScoresToBeLessThan` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_value_z_scores_to_be_less_than).</small>
+
+### Expect Column Quantile Values To Be Between
+
+Ensures that specified quantiles of a column fall within provided ranges.
+
+**Use Case**: Robustly monitors key statistics of the overall distribution, which is valuable for tracking metrics like median purchase amount or 90th percentile product ratings.
+
+```python
+gxe.expect_column_quantile_values_to_be_between(
+    column="purchase_amount",
+    quantile_ranges={
+        "quantiles": [0.5, 0.9],
+        "value_ranges": [
+            [50, 200],
+            [500, 2000]
+        ]
+    }
+)
+```
+
+<small>View `ExpectColumnQuantileValuesToBeBetween` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_quantile_values_to_be_between).</small>
+
+### Expect Column Stdev To Be Between
+
+Validates that the standard deviation of a column is within a specified range.
+
+**Use Case**: Watches for unusual changes in variance that could signal issues in data collection or processing pipelines.
+
+```python
+gxe.expect_column_stdev_to_be_between(
+    column="purchase_amount",
+    min_value=500,
+    max_value=2000
+)
+```
+
+<small>View `ExpectColumnStdevToBeBetween` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_stdev_to_be_between).</small>
+
+<br/>
+<br/>
+
+:::tip[GX tips for distribution Expectations]
+- Use the `mostly` parameter to allow for acceptable deviations in your data, providing flexibility in your validations.
+- Regularly update your reference distributions (e.g., in `ExpectColumnKlDivergenceToBeLessThan`) to reflect the most recent data patterns.
+- Combine multiple distribution Expectations to create a comprehensive validation suite that covers central tendency, dispersion, and shape of your data distribution.
+:::
+
+## Additional distribution Expectations
+
+In addition to the key distribution Expectations discussed above, GX offers a range of other Expectations that can be useful for validating and monitoring data distributions. Some examples include:
+
+- `ExpectColumnMedianToBeBetween`: Checks if the median value of a column is within a specified range. The median is the middle value when the data is sorted in ascending or descending order. This Expectation is useful for ensuring that the central tendency of the data falls within acceptable limits.
```

Comment:
Is there a reason you opted not to provide examples and Expectation Gallery links for these Expectations? I think it would be helpful to also have example usage & links for these Expectations.